pages/favorites/index.jsx

```diff
@@ -0,0 +1,21 @@
+import MovieList from "@/components/MovieList";
+
+const FavoritePages = ({ onToggleFavorite, movieInfo, movies }) => {
+  const favorites = movies?.results?.filter((movie) =>
```

Comment:
here you need to update the code to directly use movies after you updated your local API